datasets/ua/nsdc_sanctions/ua_nsdc_sanctions.yml

```
@@ -31,56 +31,56 @@ data:
+      - match: "Федеральне державне унітарне підприємство \"Ордена Трудового Червоного Прапора науково- дослідний інститут радіо\" (Федеральне державне унітарне підприємство \"Ордена Трудового Червоного Прапора російський науково-дослідний інститут радіо імені М.Й.Кривошєєва\", Федеральное государственное унитарное предприятие \"Ордена Трудового Красного Знамени российский научно-исследовательский институт радио имени М.И.Кривошеева\")"
+        value: "\"Федеральне державне унітарне підприємство \"Ордена Трудового Червоного Прапора науково- дослідний інститут радіо\""
+      - match: "Federalne derzhavne unitarne pidpryiemstvo \"Ordena Trudovoho Chervonoho Prapora naukovo- doslidnyi instytut radio\" (Federalne derzhavne unitarne pidpryiemstvo \"Ordena Trudovoho Chervonoho Prapora rosiiskyi naukovo-doslidnyi instytut radio imeni M.I.Kryvoshieieva\", Federalnoe hosudarstvennoe unytarnoe predpryiatye \"Ordena Trudovoho Krasnoho Znameny rossyiskyi nauchno-yssledovatelskyi ynstytut radyo ymeny M.Y.Kryvosheeva\")"
+        value: "\"Federalne derzhavne unitarne pidpryiemstvo \"Ordena Trudovoho Chervonoho Prapora naukovo- doslidnyi instytut radio\""
+      - match: "Holovne upravlinnia sanatorno-kurortnykh zakladiv v Avtonomnii Respublitsi Krym Derzhavnoho upravlinnia spravamy (Terytorialne upravlinnia Upravlinnia spravamy Prezydenta Rosiiskoyi Federatsiyi v Respublitsi Krym, Terrytoryalnoe upravlenye Upravlenyia delamy Prezydenta Rossyiskoi Federatsyy v Respublyke Krym)"
+        value: "\"Holovne upravlinnia sanatorno-kurortnykh zakladiv v Avtonomnii Respublitsi Krym\""
+      - match: "ДЕРЖАВНЕ ПІДПРИЄМСТВО \"ВИПРОБУВАЛЬНИЙ ЦЕНТР \"ОМЕГА\" (Филиал федерального государственного унитарного предприятия Ордена Трудового Красного Знамени Научно-исследовательский институт радио – Севастопольский «Испытательный центр «Омега», Філія федерального державного унітарного підприємства Ордена Трудового Червоного Прапора Науково-дослідний інститут радіо – Севастопольський «Випробувальний центр «Омега»)"
+        value: "\"ВИПРОБУВАЛЬНИЙ ЦЕНТР \"ОМЕГА\""
+      - match: "DERZhAVNE PIDPRYIeMSTVO \"VYPROBUVALNYI TsENTR \"OMEHA\" (Fylyal federalnoho hosudarstvennoho unytarnoho predpryiatyia Ordena Trudovoho Krasnoho Znameny Nauchno-yssledovatelskyi ynstytut radyo – Sevastopolskyi «Yspytatelnyi tsentr «Omeha», Filiia federalnoho derzhavnoho unitarnoho pidpryiemstva Ordena Trudovoho Chervonoho Prapora Naukovo-doslidnyi instytut radio – Sevastopolskyi «Vyprobuvalnyi tsentr «Omeha»)"
+        value: "\"VYPROBUVALNYI TSENTR \"OMEHA\""
+      - match: "Товариство з обмеженою відповідальністю Розрахункова небанківська кредитна організація \"Платежный центр\" (Общество с ограниченной ответственностью Расчётная небанковская кредитная организация \"Платежный центр\", Товариство з обмеженою відповідальністю Розрахункова небанківська кредитна організація \"Платіжний центр\")"
+        value: "\"Платежный центр\""
```

Ah I just realised the repetition is ukranian vs russian, right? When they differ, I think it's worth including a russian and ukranian version.

I was a bit unclear with the suggestion for the TV stations as aliases. I think just the small bit in quotes like "Платежный центр" is a bit vague so I think we should expand all the values to include the full name, just not repeating three times in the same value, so generally I think that will be

```
values:
  - name in ukranian
  - name in russian
```

```suggestion
        values:
          - "Товариство з обмеженою відповідальністю Розрахункова небанківська кредитна організація \"Платежный центр\""
          - "Общество с ограниченной ответственностью Расчётная небанковская кредитная организация \"Платежный центр\""
```